docs/maps/api-players.md

```diff
@@ -0,0 +1,257 @@
+{.section-title.accent.text-primary}
+# API Players functions Reference
+
+Functions in the `WA.players` namespace can be used to listen to nearby users.
+
+## Enabling players tracking
+
+By default, your script will not receive data from the nearby users. This is done for performance reasons.
+
+If you want to start listening to other users, the very first step is to call the `WA.players.enableTracking` method.
+
+```typescript
+WA.players.enableTracking();
+```
+
+By default, this will track players entering / leaving the "nearby zone", track players movement and track players variable.
+You can opt-out from tracking player movement (that can be resource intensive if there are hundreds of players) by using:
+
+```typescript
+WA.players.enableTracking({
```

Comment:
This makes more sense for me:

```
WA.players.tracking({
  players: true,
  movement: false,
});
```
And you can add a specific method to set one/both to true/false.

Reply:
Comment is good. `enableTracking` is a bad name because the method can be used to disable tracking.
But I don't like `tracking` only because I like my method names to be verbs.

I like using `players` instead of `trackPlayers` for options though.

I settled for:

```
WA.players.configureTracking({
  players: true,
  movement: false,
});
```